packages/gatsby-remark-autolink-headers/src/__tests__/index.js

```diff
@@ -192,7 +192,7 @@ describe(`gatsby-remark-autolink-headers`, () => {
 
 # With *Bold* {#custom-withbold}
 
-# Invalid {#this_is_italic}
+# Italic {#this_is_italic}
```

Comment:
I'm confused here:
 1. How is it `Italic` (?) the change
 2. How was it invalid? (due to underscores?)

In any case - this seems like behaviour change for this given change in assertion that doesn't just replace `invalid` with `italic`?

Reply:
Good point. I think it was a limitation of the previous parser that parsed `_is_` as italic, which made those ids invalid. The new CommonMark parser is smarter. I am not sure what exactly we can document here. One thing to do is probably to change `this_is_italic` to `this_is_not_at_all_italic`.

But open to other suggestions.

Reply:
In the [remark changelog](https://github.com/remarkjs/remark/releases/tag/13.0.0) this is probably listed under:

```
…plus a ton of other minor previous differences from CommonMark
```